test/source/mock/backend/backend-endpoints.ts

```diff
@@ -58,16 +46,36 @@ export const mockBackendEndpoints: HandlersDefinition = {
   },
 };
 
-const throwIfNotPostWithAuth = (body: unknown, req: IncomingMessage) => {
-  const parsed = body as Dict<any>;
+interface OpenId {
+  name: string;
+  email?: string;
+  email_verified?: boolean;
+}
+
+const throwIfNotPost = (req: IncomingMessage) => {
   if (!isPost(req)) {
     throw new HttpClientErr('Backend mock calls must use POST method');
   }
-  if (!parsed.account) {
-    throw new HttpAuthErr('Backend mock call missing value: account');
+};
+
+const throwIfIdTokenIsInvalid = (req: IncomingMessage) => {
+  const idToken = req.headers.authorization?.replace(/^Bearer /, '');
+  if (!idToken) {
+    throw new HttpClientErr('backend mock: Missing id_token');
   }
-  if (!parsed.uuid) {
-    throw new HttpAuthErr('Backend mock call missing value: uuid');
+  if (!oauth.isIdTokenValid(idToken)) {
+    throw new HttpAuthErr(`Could not verify mock idToken: ${idToken}`);
   }
-  return parsed;
+  return parseIdToken(idToken);
```

Comment:
I'm really not sure if we need this extra code.
As I can see, we have the code extracting claims in our mock `OauthMock.isIdTokenValid()`
We could split it in two parts (public methods): `OauthMock.extractEmailFromIdToken()` and `OauthMock.isIdTokenValid(email: string, idToken: string)`.
We won't even need to introduce the new data type `OpenId`.